### DISCUSSION TITLE
Comfyui Amd gpu problem ksampler

### DISCUSSION THREAD

**Discussion by Krueger45** [community/community#108247] [Programming Help]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/139659721/41f39c4c-9ac6-411d-9791-9dd2819979ce)
![vvgukbukjnb](https://github.com/community/community/assets/139659721/55615d6e-f129-466a-8f9b-ae3019177e2d)

**Comment by VerzatileDev** [comment]
Seems that the following is in the wrong topic, and what seems to be the question in relation to the content provided?

**Comment by queenofcorgis** [comment]
Hi @Krueger45 , we're happy you're here! You are more likely to get a useful response if you are explicit about what your problem entails, giving a few more details might help someone give you a nudge in the right direction.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
